presto-docs/src/main/sphinx/admin/tuning.rst

```diff
@@ -10,29 +10,392 @@ Config Properties
++                                          +                                          +                                    +scheduling splits. ``flat`` will try to schedule       +
++                                          +                                          +                                    +splits on the same host as the data is located by      +
++                                          +                                          +                                    +reserving 50% of the work queue for local splits.      +
++------------------------------------------+------------------------------------------+------------------------------------+-------------------------------------------------------+
++       ``task.default-concurrency``       +               ``Integer``                +               ``1``                +Default local concurrency for parallel operators. Use  +
++                                          +                                          +                                    +this as default for sessions properties:               +
++                                          +                                          +                                    +``task_hash_build_concurrency`` and                    +
++                                          +                                          +                                    +``task_aggregation_concurrency`` as well as            +
++                                          +                                          +                                    +``task.join-concurrency`` property.                    +
++------------------------------------------+------------------------------------------+------------------------------------+-------------------------------------------------------+
++        ``task.join-concurrency``         +               ``Integer``                +``task.default-concurrency`` (``1``)+Servers as default value for session property:         +
++                                          +                                          +                                    +``task_join_concurrency``. Describes local concurrency +
++                                          +                                          +                                    +for join operators. This value may be increased to     +
++                                          +                                          +                                    +perform join on worker using more then one thread.     +
++                                          +                                          +                                    +This will increase CPU utilization with the cost of    +
++                                          +                                          +                                    +increased memory usage.                                +
```

Comment:
I would drop the last sentence with memory cost, since we can not defend it (we don't know how it effects memory). I would leave it for a follow up investigation in the future.